test/configuration_unittest.cc

```diff
@@ -0,0 +1,81 @@
+#include "../src/configuration.h"
+#include "gtest/gtest.h"
+
+namespace google {
+
+class MetadataAgentConfigurationTest : public ::testing::Test {
+ public:
+  void CallParseConfiguration(MetadataAgentConfiguration& config, std::istream& stream) {
+    config.ParseConfiguration(stream);
+  }
+
+  static void SetUpTestCase() {}
```

Comment:
`SetUpTestCase` and `TearDownTestCase` are only needed for [advanced cases](https://github.com/google/googletest/blob/master/googletest/docs/AdvancedGuide.md#sharing-resources-between-tests-in-the-same-test-case), aren't they? This fixture doesn't have `static` shared state.

Reply:
Correct, removed.